src/richie/apps/core/factories.py

```diff
@@ -214,7 +214,7 @@ def subject_location(self):
         """
         return (random.randint(1, 100), random.randint(1, 100))  # nosec
 
-    # pylint: disable=no-self-use
+    # pylint: disable=no-self-use,import-outside-toplevel
```

Comment:
But doing the import at the top of the file seems to work.

Reply:
I'll admit I know nothing about this PIL module and why it needs (or does not) this special import. I'll remove it and leave it at that if it builds then